lib/puppet/type/constraint.rb

```diff
@@ -1,5 +1,5 @@
 module Puppet
-  newtype(:constraint) do
+  Puppet::Type.newtype(:constraint) do
```

Comment:
Noooooo. Then `git blame` cannot show us who wrote which line before this anymore. :stuck_out_tongue_winking_eye: